crates/typst/src/model/par.rs

```diff
@@ -382,11 +382,12 @@ impl Construct for ParLine {
 /// counter should be reset.
 #[derive(Debug, Cast, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum LineNumberingScope {
-    /// Indicates the line number counter spans the whole document, that is,
-    /// is never automatically reset.
+    /// Indicates that the line number counter spans the whole document, i.e.,
+    /// it's never automatically reset. Currently, the manual reset of
+    /// the counter is not supported.
```

Comment:
I feel like the last sentence should go on the type's own documentation.

Reply:
Fixed?